### ISSUE TITLE
GitHub (and others) support

### ISSUE THREAD

**Issue by marnovo** [IQAndreas/markdown-licenses#25]
Are these license files recognized automatically by GitHub (and other websites or tools)?

E.g. from this repo: 
<img width="206" alt="screen shot 2018-01-26 at 3 59 27 pm" src="https://user-images.githubusercontent.com/15906764/35453605-fe2e0634-02b1-11e8-858f-bf56a4af5703.png">

**Comment by vrom911** [comment]
I also wonder..

**Comment by marnovo** [comment]
@IQAndreas

**Comment by HaoZeke** [comment]
It used to work I believe. The change is recent.

**Comment by jacobwhall** [comment]
Yes, the licenses from this repo have always been properly detected by GitHub in my experience. [Here](https://github.com/jacobwhall/racket-mf2) is an example of a markdown license being detected in one of my projects.

I think markdown licenses are more preferable as ever now that GitHub has a tabbed layout on a repository's landing page, allowing visitors to click between the README, contributing guidelines, etc. The markdown-formatted license looks much nicer than plaintext in this context IMO
